examples/transformers/README.md

```diff
@@ -0,0 +1,189 @@
+ --cache_dir cache \
+ --preprocessing_num_workers 4 \
+```
+
+## Model Performance 
+To demostrate the effect that various pruning regimes and techniques can have we prune the same bert-base-uncased model to 5 different sparsities(0,80,90,95,99) using 3 pruning methodologies: oneshot(prune to desired weights before fine tune then fine tune for 1 epoch), GMP 1 epoch(prune to desired sparsity over an epoch then stabilize over another epoch), and GMP 8 epochs (prune to desired sparsity over 8 epochs then stabilize over another 2 epochs). Its worth noting that we are pruning all layers uniformly and we believe further gains can be have by targeted pruning of individual layers.
+
+| base model name       | sparsity 	| total train epochs    | prunned | one shot |pruning epochs| F1 Score 	| EM Score  |
+|-----------------------|----------	|-----------------------|---------|----------|--------------|----------	|-----------|
+| bert-base-uncased 	|0        	|1                  	|no       |no        |0            	|09.685     |3.614      |
+| bert-base-uncased 	|0        	|2                  	|no       |no        |0            	|88.002     |80.634     |
+| bert-base-uncased 	|0        	|10                 	|no       |no        |0            	|87.603     |79.130     |
+| bert-base-uncased 	|80       	|1                  	|yes      |yes       |0          	|25.141     |15.998     |
+| bert-base-uncased 	|80       	|2                   	|yes      |no        |0            	|06.068    	|00.312     |
+| bert-base-uncased 	|80       	|10                  	|yes      |no        |8          	|83.951     |74.409     |
+| bert-base-uncased 	|90       	|1                  	|yes      |yes       |0           	|16.064     |07.786     |
+| bert-base-uncased 	|90       	|2                   	|yes      |no        |0            	|64.185     |50.946     |
+| bert-base-uncased 	|90       	|10                 	|yes      |no        |8            	|79.091     |68.184     |
+| bert-base-uncased 	|95       	|1                  	|yes      |yes       |0           	|10.501     |4.929      |
+| bert-base-uncased 	|95       	|2                   	|yes      |no        |0            	|24.445     |14.437     |
+| bert-base-uncased 	|95       	|10                 	|yes      |no        |8            	|72.761  	|60.407     |
+| bert-base-uncased 	|99         |1                   	|yes      |yes       |0             |09.685     |03.614     |
+| bert-base-uncased 	|99       	|2                   	|yes      |no        |0            	|17.433     |07.871     |
+| bert-base-uncased 	|99         |10                    	|yes      |no        |8             |47.306    	|32.564     |
+
+
+## Script origin and how to integrate sparseml with other Transformers projects
+This script is based on the example BERT-QA implementation in transformers found [here](https://github.com/huggingface/transformers/blob/master/examples/question-answering/run_qa.py). 
```

Comment:
Can we get this to be a direct replacement of the run_qa.py file and have our version be a drop-in file for that?

Reply:
It does that but transformers may update the file. If you look at the bottom of the README.md there are instructions on how to drop into any transforms version